internal/config/e2eConfig.go

```diff
@@ -28,11 +28,17 @@ type E2EConfig struct {
 }
 
 type Setup struct {
-	Env       string     `yaml:"env"`
-	File      string     `yaml:"file"`
-	Manifests []Manifest `yaml:"manifests"`
-	Runs      []Run      `yaml:"runs"`
-	Timeout   int        `yaml:"timeout"`
+	Env     string `yaml:"env"`
+	File    string `yaml:"file"`
+	Steps   []Step `yaml:"steps"`
+	Timeout int    `yaml:"timeout"`
+}
+
+type Step struct {
+	Type    string `yaml:"type"`
```

Comment:
Can we remove this field and just judge from `Path != ""` / `Command != ""`, and throw an error when `Path == "" && Command == ""`?